### ISSUE TITLE
[Suggestion] Thunar keybind / F4 to Open terminal here

### ISSUE THREAD

**Issue by samwhelp** [BunsenLabs/bunsen-configs#127]
# [Suggestion] Thunar keybind / F4 to Open terminal here

## .config/Thunar/uca.xml

* [.config/Thunar/uca.xml](https://github.com/BunsenLabs/bunsen-configs/blob/boron/skel/.config/Thunar/uca.xml#L11-L18)

> Code Snippet / orginal 

``` xml
  <action>
    <icon>utilities-terminal</icon>
    <name>Open terminal here</name>
    <command>sh -c &apos;cd &quot;$1&quot;; x-terminal-emulator&apos; _ %f</command>
    <description />
    <patterns>*</patterns>
    <directories />
  </action>
```

add one line `<unique-id>Action_Open_Terminal_Here</unique-id>`

> Code Snippet / adjusted

``` xml
  <action>
    <icon>utilities-terminal</icon>
    <name>Open terminal here</name>
    <unique-id>Action_Open_Terminal_Here</unique-id>
    <command>sh -c &apos;cd &quot;$1&quot;; x-terminal-emulator&apos; _ %f</command>
    <description />
    <patterns>*</patterns>
    <directories />
  </action>
``` 

## ~/.config/Thunar/accels.scm

add one file `~/.config/Thunar/accels.scm`

content:

```
(gtk_accel_path "<Actions>/ThunarActions/uca-action-Action_Open_Terminal_Here" "F4")
```

I created a [pull request 126](https://github.com/BunsenLabs/bunsen-configs/pull/126) for this suggestion.

**Comment by johnraff** [comment]
Hi @samwhelp thank you for this useful suggestion! Your pull request #126 looks solid, but before I merge it I'd just like to ask: where did your choice of F4 for the keybind come from? Are there other file managers which use the same keybind to open a terminal?

**Comment by samwhelp** [comment]
Thanks for reply.

These file manager as follow, using `F4` to open terminal here

* [pcmanfm](https://packages.debian.org/stable/pcmanfm)
* [pcmanfm-qt](https://packages.debian.org/stable/pcmanfm-qt)
* [dolphin](https://packages.debian.org/stable/dolphin)

> dolphin use `F4` to open embedded terminal, use `Shift + F4` to open new terminal (konsole)

**Comment by johnraff** [comment]
Fair enough. Merged #126 with thanks.
